source/includes/fundamentals/code-snippets/AggTour.java

```diff
@@ -48,29 +51,37 @@ public static void main(String[] args) {
         // begin aggregation one
         collection.aggregate(
             Arrays.asList(
+                // Creates instructions to match documents that contain the "Bakery" category
```

Comment:
Issue:

From our [code comment style guide](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit): "Avoid comments within a multi-line variable assignment or method call."

Suggestion:

I think the comments about the aggregate stages should be relocated to before the collection.aggregate() call and combined.

"Creates an aggregation pipeline that matches documents... and groups them by..."